hpe3parclient/client.py

```diff
@@ -4439,3 +4439,141 @@ def targetInRemoteCopyGroupExists(
         except Exception:
             pass
         return False
+
+    def check_response(self, resp):
+        for r in resp:
+            if 'error' in str.lower(r) or 'invalid' in str.lower(r) or 'the schedule format' in str.lower(r):
+                err_resp = r.strip()
+                return err_resp
+
+    def createSchedule(self, schedule_name, task, taskfreq='@hourly'):
```

Comment:
Why are setting default value? Why can we not have the frequency as a mandatory parameter?

Reply:
removed